### DISCUSSION TITLE
Can't install and run Ubuntu

### DISCUSSION THREAD

**Discussion by TruthfulEdict1** [utmapp/UTM#2597] [UTM Help (macOS)] [answered]
I can't install Ubuntu. It doesn't work as the guide describes, so I went through the troubleshooting at the bottom of the page here:

https://mac.getutm.app/gallery/ubuntu-20-04

When I type the suggested command fs0:\efi\boot\bootaa44.efi into the shell, all I get is the following error message:

"is not recognized as an internal or external command, operable program, or script file"

I'm not sure what to search for to find existing threads about this, and I'm new to Github so not sure how to browse through the pinned posts either, sorry. 

**Configuration**
* UTM Version: Not sure how to find this, but I just downloaded it today, so the latest version off of the UTM site. 
* OS Version: Big Sur 11.4
* Apple Silicon 

[debug.log](https://github.com/utmapp/UTM/files/6572891/debug.log)

**Comment by winexe0** [comment]
The command is fs0:\efi\boot\bootaa64.efi

**Reply by s892307** [reply]
I'm having this exact same error, and have typed the command correctly.  Is there anything else I can try?

I'm running an M1 chip, and am trying to install with an Ubuntu 20.04 ISO.

```
Shell> fs0:\efi\boot\bootaa64.efi

'fs0:\efi\boot\bootaa64.efi' is not recognized as an internal or external command, operable program, or script file
```

**Reply by conath** [reply]
Try this: After you type `fs0:`, press return/enter. Then type the rest. @TruthfulEdict1

**Reply by yimingshan666** [reply]
After I type `fs0:` , press return/enter. Then I type in `\efi\boot\bootaa64.efi` , Error still displayed, I also tried `efi\boot\bootaa64.efi` @conath

**Comment by TruthfulEdict1** [comment]
> The command is fs0:\efi\boot\bootaa64.efi

Thank you, yes I see I typed "bootaa44" instead of "bootaa64" in my post, but I tried again with the correct command and still get the same message.

**Comment by hungtcs** [comment]
I can install normally after changing the display to a character interface.

**Comment by yimingshan666** [comment]
I have the same problem. Did you solve it?

**Comment by conath** [comment]
For what it's worth, I was able to install Ubuntu Server for ARM 20.04 LTS just fine following the guide, on my M1 Mac. [https://ubuntu.com/download/server/arm](Download Ubuntu Server for ARM here.)

**Comment by yongmingyang** [comment, marked as answer]
Hi guys, I had the same issue. I realized that it wasn't working because I was using Ubuntu desktop's iso instead of the server's iso which you can download from here: https://ubuntu.com/download/server/arm

Hope that helps

**Comment by clocky** [comment]
Hit the same problem, and it was as dumb as me accidentally downloading the x64 ISO (`ubuntu-20.04.3-live-server-amd64`) instead of the correct ARM one (`ubuntu-20.04.3-live-server-arm64`). 

The `bootaa64.efi` file doesn't exist on the x64 ISO.

**Comment by ShivamPansy** [comment]
Can we not install `ubuntu-20.04.3-live-server-amd64` ?  I have been trying to install it and it given me error like 
`/dev/loop2: can't open blockdev`
